components/generic/SimilarItems.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section class="row py-4 px-2 bg-white">
+    <b-col>
+      <b-card-group
+        class="card-deck-4-cols similar-items"
+        deck
+      >
+        <ContentCard
+          v-for="(item, index) in items"
+          :key="index"
+          :title="item.name"
+          :texts="texts"
+          :url="item.identifier"
+          :thumbnail="item.thumbnail"
+        />
+      </b-card-group>
+    </b-col>
+  </section>
+</template>
+
+<script>
+  import ContentCard from './ContentCard';
+
+  export default {
+    name: 'SimilarItems',
+
+    components: {
+      ContentCard
+    },
+
+    props: {
+      items: {
```

Comment:
This needs a `linkGen` prop too. It should default to `(identifier) => identifier`, and it should be called on the ContentCard component iterator, like: `:url="linkGen(identifier)"`.

When we wire this all up, the parent component will pass in an actual link generation function.

Reply:
After further consideration we decided just to hard-code the item page route in the component.